src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -65,6 +65,16 @@ public function signupAction(Request $request)
                 $command = new AddUserCommand($targetUser, $encryptedPassword);
                 $this->sendCommand($command);
 
+                // email the new user
+                $email = $targetUser->getUsername();
+                $message = (new \Swift_Message('Hello Email'))
+                ->setFrom('send@example.com')
```

Comment:
The `from` email address still needs to be configurable.

Reply:
This was always causing trouble during testing so left it hard coded for now